### PR TITLE
initial content for the buddy guide and learning resources

### PR DESCRIPTION
**Pull request by JameelB**
# What
Initial content for the buddy guide and learning resources. The initial content of the buddy guide is based on the buddy guide in our help repo.

### CODE REVIEW

File: buddy-guide.md [hunk 1]
```diff
@@ -7,24 +7,20 @@
...
 # Buddy Guide
-
-
+As part of Red Hat's internship programme, each new hire will be allocated a buddy. A buddy is an experienced team member who will be able to provide training, instruction and leadership for the student. This is a guide for what is expected of you as a buddy to ensure that the student have a pleasant experience during their internship.
 
 
 # Before the new student starts
-
-
+- Ensure you are familiar with the student you are paired up with. Read their CV, learn something about them.
```
Comment (laurafitzgerald) [inline buddy-guide.md]: @JameelB We decided that we would only include content in this repo that would be used by the student/intern. This is more internal stuff for ourselves

Comment (damienomurchu) [reply buddy-guide.md]: Probably a need for 2 docs - a buddy guide which would outline the duties/ responsibilities of a buddy, and a student guide, which would include a section on what the student could expect from their buddy.

Comment (laurafitzgerald) [reply buddy-guide.md]: @damienomurchu yes agreed. it would suit the scope of this repo to update this to `what to expect from your buddy`

**Review by damienomurchu — APPROVED** [review]
Looks good to me & meets remit of initial content for buddy-guide and learning resources. Happy to see it merged & further content added in subsequent PRs

**Comment by laurafitzgerald** [comment]
@JameelB looks great. Also happy to get this merged. If there is no related JIRA lets create one to track and make visible what work is being done.

**Comment by JameelB** [comment]
@laurafitzgerald @damienomurchu I will be moving this content to the buddy-guide [here](https://github.com/feedhenry/student-help-guide/blob/master/first-day/buddy/index.html) This issue is being tracked by [FH-4075](https://issues.jboss.org/browse/FH-4075).

**Comment by laurafitzgerald** [comment]
@JameelB great :)

**Comment by JameelB** [comment]
Closing as work has been moved to https://github.com/feedhenry/student-help-guide/pull/17

**Pull request closed without merging**